src/AppDirs.vala

```diff
@@ -326,6 +326,9 @@ class AppDirs {
     public static File get_facedetect_bin() {
         const string filename = "shotwell-facedetect";
         File f = File.new_for_path(AppDirs.get_exec_dir().get_path() + "/facedetect/" + filename);
+        if (!f.query_exists()) {//testing meson builddir
+            f = File.new_for_path(AppDirs.get_exec_dir().get_path() + "/../facedetect/" + filename);
+        }
```

Comment:
Please use the File api to navigate the tree, eg.g. get_exec_dir().get_parent().get_child("facedetect") etc. or resolve_relative_path(). I know the rest of the code doesn't do it, but we should start to get a bit nicer here